go/adbc/adbc.go

```diff
@@ -237,6 +237,7 @@ const (
 	OptionKeyProgress                 = "adbc.statement.exec.progress"
 	OptionKeyMaxProgress              = "adbc.statement.exec.max_progress"
 	OptionKeyIngestTargetTable        = "adbc.ingest.target_table"
+	OptionKeyIngestTemporary          = "adbc.ingest.temporary"
```

Comment:
We haven't technically added these options to the 'specification' interface

Reply:
@zeroshade is there any way to mark things 'experimental'?

Reply:
you can use godoc to put a comment in there

```go
        // This option is experimental
        OptionKeyIngestTemporary = "adbc.ingest.temporary"
```

If you want it to get bolded or otherwise you can use `// # This option is experimental` and it'll show up as a header in the docs.